pkg/azurediskplugin/main.go

```diff
@@ -70,6 +70,7 @@ var (
 	enableAzureClientAttachDetachRateLimiter = flag.Bool("enable-attach-detach-rate-limiter", consts.DefaultEnableAzureClientAttachDetachRateLimiter, "azure client rate limiter for attach/detach operations (true by default)")
 	azureClientAttachDetachRateLimiterQPS    = flag.Float64("attach-detach-rate-limiter-qps", consts.DefaultAzureClientAttachDetachRateLimiterQPS, "QPS for azure client rate limiter for attach/detach operations (1.3 by default)")
 	azureClientAttachDetachRateLimiterBucket = flag.Int("attach-detach-rate-limiter-bucket", consts.DefaultAzureClientAttachDetachRateLimiterBucket, "bucket for azure client rate limiter for attach/detach operations (80 by default)")
+	azureClientAttachDetachBatchInitialDelay = flag.Duration("attach-detach-batch-initial-delay", consts.DefaultAzureClientAttachDetachBatchInitialDelay, "initial batch processing start delay for attach/detach operations (1s by default)")
```

Comment:
// A value of `0` indicates to use the default initial delay of 1s.
// A negative value indicates no initial delay.
// A positive value determines the initial delay in milliseconds.

Should this information be enclosed somewhere?